src/content/en/tools/chrome-devtools/accessibility/navigation.md

```diff
@@ -0,0 +1,348 @@
+
+Note: Be sure to read through [the styles tab keyboard
+reference](/web/tools/chrome-devtools/shortcuts#styles) for additional Style tab
+shortcuts.
+
+##### Known Issues
+
+- If you use the **filter** editable text field you will no longer be able to
+  navigate the list of styles.
+
+#### Toggle element state
+
+To toggle an element's state, such as its active or focused state, navigate to
+the Style tab and press `Tab` until the **Toggle Element State** button has
+focus. Press `Enter` to expand the collection of element states.
+
+The element states are presented as a group of checkboxes. Press `Tab` until the
+first state, `:active`, has focus. Then press `Space` to enable it. If the
+currently selected element in the DOM tree has an `:active` style, it will now
+be applied. Continue pressing `Tab` to explore all of the available states.
+
+#### Add an exiting class
+
+Adjacent to the Toggle Element State button is the **Element Classes** button.
+Move focus to it by pressing `Tab` and press `Enter`. Focus will move into an
+edit text field labeled "Add new class".
+
+The Element Classes button is primarily used for adding existing classes to an
+element. For example, if your stylesheet contained a helper class named
+`.clearfix` you could press `.` inside of the edit text field to see a
+suggestion list of classes and use the `Down Arrow` to find the `.clearfix`
+suggestion. Or type the class name out yourself and press `Enter` to apply it.
+
+#### Add a new style rule
+
+Adjacent to the Element Classes button is the **New Style Rule** button. Move
+focus to it by pressing `Tab` and press `Enter`. Focus will move into an
+editable text field inside of the style inspector. The initial text content of
+the field will be the tag name of the element that is selected in the DOM tree.
+You may type any class name you want into this field and then press tab to
+assign CSS properties to it.
+
+### Computed tab
+
+With focus on the [**Computed**
+tab](/web/tools/chrome-devtools/css/reference#computed), press `Tab` to move
+focus inside and explore its contents.
+
+Within the Computed tab there are controls for exploring which CSS properties
+are actually applied to an element in order of specificity.
+
+#### Explore all computed styles
+
+Press `Tab` until you reach the collection of computed styles. These are
+presented as an ARIA `tree`. Expanding a listbox will reveal which CSS selectors
+are applying the computed style. These selectors are organized by specificity.
+
+A screen reader will announce the computed value, which CSS selector is
+currently matching, the filename of the stylesheet that contains the selector,
+and the line number for the selector.
+
+#### Known issues
+
+- If you use the filter box you can no longer inspect styles.
+
+### Event listeners tab
+
+From within the **Elements** panel you may inspect the event listeners applied
+to an element using the **Event Listeners** tab. // TODO: Is this tab
```

Comment:
@kaycebasques 	left a todo. Is there an official doc that explains this tab anywhere? If so i was going to link to it.